app/api/business/brief_overview_business.py

```diff
@@ -1,8 +1,69 @@
 from flask_login import current_user
+
 from app.api.business.agreement_business import use_old_work_order_creator
+from app.api.helpers import abort
+from app.api.services import (audit_service, audit_types,
+                              brief_responses_service, briefs)
+from app.emails import send_opportunity_closed_early_email
+from app.tasks import publish_tasks
+from app.tasks.s3 import create_responses_zip
 from app.validation import get_sections as get_validation_sections
 
 
+def can_close_opportunity_early(brief):
+    seller_selector = brief.data.get('sellerSelector', '')
+    invited_sellers = brief.data.get('sellers', {}).keys()
+    number_of_candidates = int(brief.data.get('numberOfSuppliers', 0))
+
+    if brief.status == 'live' and len(invited_sellers) == 1 and (
+        (brief.lot.slug in ['rfx', 'training2'] and seller_selector == 'oneSeller') or
+        (brief.lot.slug == 'specialist' and seller_selector == 'someSellers')
```

Comment:
This thoroughness is not a bad thing but I think just confirming there was only 1 invited seller is probably enough?

Reply:
I'd prefer to leave this as is as it helps describe exactly when an opportunity can be closed.